### ISSUE TITLE
Doesn't understand Chinese

### ISSUE THREAD

**Issue by scythetleppo** [argosopentech/argos-translate#275]
**Hello.**
![image](https://user-images.githubusercontent.com/99230615/173962594-7d485359-9f01-4f84-8eb9-3ea3ec4ed992.png)

**How are you?**
![image](https://user-images.githubusercontent.com/99230615/173962679-e6742666-e4ba-4cb5-9289-e342b037dbb9.png)

**I'm too/very tired**
![image](https://user-images.githubusercontent.com/99230615/173962742-a1ce196a-32cd-4868-8466-bc95e3d8b948.png)

**Comment by DSPerson** [comment]
Yes, you are right. idiot

**Comment by PJ-Finlay** [comment]
Yes the Chinese translations aren't very good. I think the root cause is that there isn't very much data available for Chinese.

**Comment by rafael3382** [comment]
Looks like there's much data for Chinese.
https://github.com/Helsinki-NLP/Tatoeba-Challenge/blob/master/data/README-v2021-08-07.md

Can someone train an argos package with it please? I really need good Chinese to English translation

**Comment by pierotofy** [comment]
Maybe you can help us train a better Chinese model @rafael3382 see https://github.com/argosopentech/argos-train

**Comment by PJ-Finlay** [comment]
The Chinese model was updated recently hopefully the new one is better. 

https://community.libretranslate.com/t/improving-chinese-translations/364/

If we can find more data we could retrain again too.

**Comment by BackMountainDevil** [comment]
still bad. How many GPU cards need if I want to train it?

**Comment by mkunz7** [comment]
https://huggingface.co/Helsinki-NLP/opus-mt-zh-en does a pretty good job, I wonder if we can use that.

```# pip install transformers
# pip install torch
# pip install sentencepiece
# pip install sacremoses

from transformers import MarianMTModel, MarianTokenizer

def chinese_to_english(text):
    model_name = 'Helsinki-NLP/opus-mt-zh-en'
    model = MarianMTModel.from_pretrained(model_name)
    tokenizer = MarianTokenizer.from_pretrained(model_name)

    # Tokenize the text
    tokenized_text = tokenizer.encode(text, return_tensors="pt")

    # Translate the tokenized text
    translated_tokens = model.generate(tokenized_text)

    # Decode the translated tokens to a string
    translated_text = tokenizer.decode(translated_tokens[0], skip_special_tokens=True)
    return translated_text

if __name__ == "__main__":
    chinese_text = input("Enter Chinese text: ")
    translated_text = chinese_to_english(chinese_text)
    print(f"Translated Text: {translated_text}")```

**Comment by pierotofy** [comment]
New Chinese simplified/traditional models (from OPUS-MT) are up:

https://libretranslate.com/?source=zh&target=en&q=%E4%BD%A0%E5%A5%BD

**Comment by gkielian** [comment]
Thanks @pierotofy! After reinstalling not only `zh` but also `pl`  is now working great : )
